saleor/graphql/account/mutations/account.py

```diff
@@ -66,17 +64,30 @@ def clean_input(cls, info, instance, data, input_cls=None):
             return super().clean_input(info, instance, data, input_cls=None)
         elif not data.get("redirect_url"):
             raise ValidationError(
-                {"redirect_url": "This field is required."},
-                code=AccountErrorCode.INVALID,
+                {
+                    "redirect_url": ValidationError(
+                        "This field is required.", code=AccountErrorCode.REQUIRED
+                    )
+                }
             )
 
         try:
             validate_storefront_url(data["redirect_url"])
         except ValidationError as error:
             raise ValidationError(
-                {"redirect_url": error}, code=AccountErrorCode.INVALID
+                {
+                    "redirect_url": ValidationError(
+                        error.message, code=AccountErrorCode.INVALID
+                    )
+                }
             )
 
+        password = data["password"]
+        try:
+            password_validation.validate_password(password, instance)
```

Comment:
`(password,` can be just `(data["password"],` no need for the upper var as it's value is self-explainatory